cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1650,6 +1671,124 @@ function Install-Pigz {
   }
 }
 
+# Node Problem Detector Resources
+$NPD_SERVICE = "node-problem-detector"
+$DEFAULT_NPD_VERSION = '0.8.7-17-g06732e5'
+# DO NOT SUBMIT - Remove the jeremyje-public from RELEASE_PATH and use an NPD release that includes Windows support.
+$DEFAULT_NPD_RELEASE_PATH = 'https://storage.googleapis.com/jeremyje-public/kubernetes-release'
```

Comment:
do you want to update these paths to nodeproblemdetector-artifacts?

Reply:
It's best to wait for NPD to update their build first.

Reply:
NPD 0.8.8 has been released with Windows support! Updated the references.